hw/ip/otbn/doc/dv_plan/index.md

```diff
@@ -24,83 +16,124 @@ applicable. Once done, remove this comment before making a PR. -->
+
+</div>
+
+The pre-DV testing doesn't address external stimuli like resets or TileLink-based register accesses.
+These are driven by specialised test sequences, described below.
+
 #### Test sequences
-All test sequences reside in `hw/ip/otbn/dv/uvm/env/seq_lib`.
-The `otbn_base_vseq` virtual sequence is extended from `cip_base_vseq` and serves as a starting point.
-All test sequences are extended from `otbn_base_vseq`.
-It provides commonly used handles, variables, functions and tasks that the test sequences can simple use / call.
-Some of the most commonly used tasks / functions are as follows:
-* task 1:
-* task 2:
+
+The test sequences can be found in `hw/ip/otbn/dv/uvm/env/seq_lib`.
+The basic test sequence (`otbn_base_vseq`) loads the instruction stream from a randomly chosen binary (see above), configures OTBN and then lets it run to completion.
+
+More specialized sequences include things like multiple runs, register accesses during operation (which should fail) and memory corruption.
+We also check things like the correct operation of the interrupt registers.
 
 #### Functional coverage
-To ensure high quality constrained random stimulus, it is necessary to develop a functional coverage model.
-The following covergroups have been developed to prove that the test intent has been adequately met:
-* cg1:
-* cg2:
+
+<div class="bd-callout bd-callout-warning">
+
+**TODO**: Functional coverage points are not yet defined.
+
+</div>
+
 
 ### Self-checking strategy
 #### Scoreboard
-The `otbn_scoreboard` is primarily used for end to end checking.
-It creates the following analysis ports to retrieve the data monitored by corresponding interface agents:
-* analysis port1:
-* analysis port2:
-<!-- explain inputs monitored, flow of data and outputs checked -->
+
+If any of the consistency checks in `otbn_core_model` that compare the RTL with the ISS fail, the module raises an error signal, bound as `err_if`.
+This is the most fundamental check in `otbn_scoreboard`: that signal should never go high.
+
+The scoreboard also defines some other checks, to ensure that interrupt and idle signals are high at the expected times.
 
 #### Assertions
-* TLUL assertions: The `tb/otbn_bind.sv` binds the `tlul_assert` [assertions]({{< relref "hw/ip/tlul/doc/TlulProtocolChecker.md" >}}) to the IP to ensure TileLink interface protocol compliance.
-* Unknown checks on DUT outputs: The RTL has assertions to ensure all outputs are initialized to known values after coming out of reset.
-* assert prop 1:
-* assert prop 2:
+
+Core TLUL protocol assertions are checked by binding the [TL-UL protocol checker]({{< relref "hw/ip/tlul/doc/TlulProtocolChecker.md" >}}) into the design.
+
+All outputs are also checked for `'X` values by assertions in `tb/otbn_bind.sv`.
```

Comment:
This is actually done in the RTL...

Reply:
Oh, so it is! I hadn't realised that, thanks.